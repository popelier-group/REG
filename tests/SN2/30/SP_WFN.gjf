# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -1.47608000   -0.00008600    0.00015400
 H                 -1.14647600   -0.87495000   -0.54517700
 H                 -1.14636100    0.90960500   -0.48477700
 H                 -1.14662300   -0.03497400    1.03052000
 Br                 1.74491500    0.00001500   -0.00001600
 Cl                -3.33229200    0.00000400   -0.00003400

frame.wfn
