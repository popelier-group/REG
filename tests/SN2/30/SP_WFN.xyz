
6

C       -1.340596    0.012817    0.130738
H       -0.932858   -0.536392    0.969506
H       -1.105262   -0.474359   -0.806646
H       -0.999586    1.040006    0.133506
Br        1.865057   -0.017647   -0.181789
Cl       -3.187983    0.030087    0.310695
