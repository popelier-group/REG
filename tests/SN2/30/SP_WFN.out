 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.30/Gau-75566.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.30/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/l1.exe PID=     75567.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -1.47608  -0.00009   0.00015 
 H                    -1.14648  -0.87495  -0.54518 
 H                    -1.14636   0.90961  -0.48478 
 H                    -1.14662  -0.03497   1.03052 
 Br                    1.74492   0.00002  -0.00002 
 Cl                   -3.33229   0.       -0.00003 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -1.476080   -0.000086    0.000154
      2          1           0       -1.146476   -0.874950   -0.545177
      3          1           0       -1.146361    0.909605   -0.484777
      4          1           0       -1.146623   -0.034974    1.030520
      5         35           0        1.744915    0.000015   -0.000016
      6         17           0       -3.332292    0.000004   -0.000034
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.082318   0.000000
     3  H    1.082317   1.785577   0.000000
     4  H    1.082318   1.785604   1.785596   0.000000
     5  Br   3.220995   3.069675   3.069499   3.069889   0.000000
     6  Cl   1.856212   2.416716   2.416742   2.416695   5.077207
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -1.340596    0.012817    0.130738
      2          1           0       -0.932858   -0.536392    0.969506
      3          1           0       -1.105262   -0.474359   -0.806646
      4          1           0       -0.999586    1.040006    0.133506
      5         35           0        1.865057   -0.017647   -0.181789
      6         17           0       -3.187983    0.030087    0.310695
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         157.2777748           0.7653646           0.7653645
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       164.5517833677 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  2.76D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147525.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(RB3LYP) =  -3074.40294848     A.U. after   11 cycles
            NFock= 11  Conv=0.84D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.55436-101.41403 -62.20735 -56.02516 -56.02508
 Alpha  occ. eigenvalues --  -56.02508 -10.08683  -9.32824  -8.41602  -7.09203
 Alpha  occ. eigenvalues --   -7.08344  -7.08344  -6.23909  -6.23875  -6.23874
 Alpha  occ. eigenvalues --   -2.35146  -2.35135  -2.35135  -2.35108  -2.35108
 Alpha  occ. eigenvalues --   -0.71346  -0.54588  -0.48751  -0.31036  -0.31035
 Alpha  occ. eigenvalues --   -0.27614  -0.17904  -0.17904  -0.05304  -0.05159
 Alpha  occ. eigenvalues --   -0.05159
 Alpha virt. eigenvalues --    0.10799   0.12074   0.14709   0.14710   0.17445
 Alpha virt. eigenvalues --    0.18129   0.19491   0.19491   0.20618   0.20619
 Alpha virt. eigenvalues --    0.21603   0.22295   0.22295   0.24743   0.28034
 Alpha virt. eigenvalues --    0.30481   0.30481   0.36340   0.40279   0.53258
 Alpha virt. eigenvalues --    0.53258   0.55099   0.58373   0.58373   0.61154
 Alpha virt. eigenvalues --    0.62061   0.62061   0.63979   0.76235   0.76235
 Alpha virt. eigenvalues --    0.79451   0.84266   0.87993   0.87993   0.89239
 Alpha virt. eigenvalues --    0.89240   0.94753   0.96329   0.96330   1.00700
 Alpha virt. eigenvalues --    1.00700   1.02571   1.06217   1.25874   1.25874
 Alpha virt. eigenvalues --    1.26441   1.56673   1.67292   1.67292   1.84548
 Alpha virt. eigenvalues --    1.84548   1.86297   2.10167   2.21634   2.21637
 Alpha virt. eigenvalues --    2.26925   2.26926   2.27832   2.45305   2.45305
 Alpha virt. eigenvalues --    2.55458   2.58889   2.63755   2.63757   2.66499
 Alpha virt. eigenvalues --    2.66501   3.08762   3.28777   3.86755   3.86757
 Alpha virt. eigenvalues --    4.39713   4.39716   4.40355   4.40356   4.49656
 Alpha virt. eigenvalues --    6.75772   7.80391   7.80392   7.85171   9.83706
 Alpha virt. eigenvalues --   24.02725  25.98319  25.98320  26.48452  48.07213
 Alpha virt. eigenvalues --  215.82269 290.01213 290.01213 290.056371020.82812
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.616203   0.401094   0.401091   0.401099  -0.048799  -0.121240
     2  H    0.401094   0.518224  -0.037887  -0.037886   0.002807  -0.038952
     3  H    0.401091  -0.037887   0.518197  -0.037886   0.002806  -0.038935
     4  H    0.401099  -0.037886  -0.037886   0.518254   0.002809  -0.038972
     5  Br  -0.048799   0.002807   0.002806   0.002809  35.921876   0.048018
     6  Cl  -0.121240  -0.038952  -0.038935  -0.038972   0.048018  17.188910
 Mulliken charges:
               1
     1  C   -0.649449
     2  H    0.192600
     3  H    0.192614
     4  H    0.192582
     5  Br  -0.929517
     6  Cl   0.001170
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C   -0.071653
     5  Br  -0.929517
     6  Cl   0.001170
 Electronic spatial extent (au):  <R**2>=           1246.1455
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=             -4.7459    Y=              0.0452    Z=              0.4627  Tot=              4.7686
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -76.1306   YY=            -44.4765   ZZ=            -44.7744
   XY=              0.2989   XZ=              3.0853   YZ=             -0.0291
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -21.0034   YY=             10.6507   ZZ=             10.3527
   XY=              0.2989   XZ=              3.0853   YZ=             -0.0291
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=             44.4246  YYY=              0.6188  ZZZ=             -2.6077  XYY=              8.5818
  XXY=             -0.2653  XXZ=             -2.6626  XZZ=              8.7250  YZZ=             -0.9361
  YYZ=             -0.7458  XYZ=             -0.0651
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=          -1658.1900 YYYY=            -76.6570 ZZZZ=            -90.1710 XXXY=              8.2250
 XXXZ=             84.5689 YYYX=              5.9563 YYYZ=             -0.5780 ZZZX=             69.8116
 ZZZY=             -0.8901 XXYY=           -263.5969 XXZZ=           -267.2213 YYZZ=            -27.8016
 XXYZ=             -0.2100 YYXZ=             23.1174 ZZXY=              3.0352
 N-N= 1.645517833677D+02 E-N=-7.682387231142D+03  KE= 3.070017516133D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE729\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\ZZALSPA2\11-No
 v-2020\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\
 C,0,-1.47608,-0.000086,0.000154\H,0,-1.146476,-0.87495,-0.545177\H,0,-
 1.146361,0.909605,-0.484777\H,0,-1.146623,-0.034974,1.03052\Br,0,1.744
 915,0.000015,-0.000016\Cl,0,-3.332292,0.000004,-0.000034\\Version=ES64
 L-G16RevC.01\State=1-A\HF=-3074.4029485\RMSD=8.442e-09\Dipole=-1.87611
 14,-0.0000735,0.0001209\Quadrupole=-15.8411735,7.9205701,7.9206034,0.0
 000208,-0.000201,-0.00001\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 DO YOU KNOW THAT THE LETTERS IN "EPISCOPAL", WHEN
 REARRANGED, SPELL "PEPSI COLA"?
 Job cpu time:       0 days  0 hours  0 minutes 14.0 seconds.
 Elapsed time:       0 days  0 hours  0 minutes  7.3 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 19:02:13 2020.
